lib/optimizely/odp/odp_manager.rb

```diff
@@ -32,7 +32,16 @@ class OdpManager
     ODP_CONFIG_STATE = Helpers::Constants::ODP_CONFIG_STATE
 
     # update_odp_config must be called to complete initialization
-    def initialize(disable:, segments_cache: nil, segment_manager: nil, event_manager: nil, fetch_segments_timeout: nil, odp_event_timeout: nil, logger: nil)
+    def initialize(
+      disable:,
+      segments_cache: nil,
+      segment_manager: nil,
+      event_manager: nil,
+      fetch_segments_timeout: nil,
+      odp_event_timeout: nil,
+      odp_flush_interval: nil,
+      logger: nil
+    )
```

Comment:
ah, here it's just reformatting into a column shape and added odp_event_timeout and odp_flush_interval...